src/auto-evo/selection_pressure/MetabolicStabilityPressure.cs

```diff
@@ -41,8 +41,14 @@ public override float Score(Species species, Patch patch, SimulationCache cache)
 
         if (energyBalance.FinalBalanceStationary > 0)
         {
-            // Punish microbes that can't move continuously severely
-            return 0.25f;
+            // Only punish non-sessile species for not being able to move continuously
+            if (microbeSpecies.Behaviour.Activity > 0)
+            {
+                return 0.25f;
+            }
+
+            // Sessile species get full score if they can maintain energy while stationary
+            return 1.0f;
```

Comment:
```suggestion
            // Only punish non-sessile species for not being able to move continuously
            return 1.0 - microbeSpecies.Behaviour.Activity / Constants.MAX_SPECIES_ACTIVITY;
```
This hasn't actually been tested, but it should scale the score based on the activity so it isn't just a huge drop off.